lib/ts/recipe/openid/types.ts

```diff
@@ -0,0 +1,86 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import OverrideableBuilder from "supertokens-js-override";
+import { BaseRequest, BaseResponse } from "../../framework";
+import NormalisedURLDomain from "../../normalisedURLDomain";
+import NormalisedURLPath from "../../normalisedURLPath";
+import { RecipeInterface as JWTRecipeInterface, APIInterface as JWTAPIInterface } from "../jwt/types";
+
+export type TypeInput = {
+    issuer: string;
```

Comment:
This should be optional here. By default, it should be equal to `appInfo.apiDomain.getAsStringDangerous() + appInfo.apiBasePath.getAsStringDangerous()`

Reply:
The default should be 
```js
supertokens.appInfo.apiDomain.getAsStringDangerous() +
                supertokens.appInfo.apiGatewayPath.getAsStringDangerous() +
                supertokens.appInfo.apiBasePath.getAsStringDangerous()
```

Reply:
done, as discussed since the gateway path and base path get merged during `supertokens.init` using

```typescript
appInfo.apiDomain + appInfo.apiBasePath
```

for the default issuer